src/utils/rest-api.js

```diff
@@ -111,6 +111,37 @@ export function getVoltageLevelSingleLineDiagram(
     );
 }
 
+export function getSubstationSingleLineDiagram(
+    studyName,
+    userId,
+    substationId,
+    useName,
+    centerLabel,
+    diagonalLabel,
+    substationLayout
+) {
+    console.info(
+        `Getting url of substation diagram '${substationId}' of study '${studyName}' of user '${userId}'...`
+    );
+    return (
+        PREFIX_STUDY_QUERIES +
```

Comment:
Coud you use the new getStudyUrl function?

Reply:
Done